### PR TITLE
Upgrade Webots to R2022a

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Kompajliraj sa:
```
colcon build --packages-ignore mep3_driver
```

Navigacija je prilagodjena Galactic-u.

Za adaptaciju ros2_control na Galactic mozes pogledati:
https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2_control/src/Ros2ControlSystem.cpp
(trazi `#if FOXY`)

---

Instalacija Galactic-a:
https://docs.ros.org/en/galactic/Installation.html

Prebaciti na `ros-testing` (barem privremeno):
https://docs.ros.org/en/rolling/Installation/Testing.html#debian-testing-repository

Azurirati Webots na R2022a:
https://github.com/cyberbotics/webots/releases/tag/R2022a

### CODE REVIEW

**Review by angstrem98 — DISMISSED** [review]
Hvala :)

File: install.bash [hunk 1]
```diff
@@ -23,38 +23,28 @@ export LANG=en_US.UTF-8
...
 
 case "${TARGET_HARDWARE}" in
-    'pc') sudo apt install -y ros-foxy-desktop python3-rosdep;;
-    'rpi') sudo apt install -y ros-foxy-ros-base python3-rosdep;;
+    'pc') sudo apt install -y ros-galactic-desktop python3-rosdep;;
+    'rpi') sudo apt install -y ros-galactic-ros-base python3-rosdep;;
 esac
 
 # Install apt dependencies
 sudo apt install -y git git-lfs build-essential python3-argcomplete python3-colcon-common-extensions
```
Comment (angstrem98) [inline install.bash:35]: Izbaciti git-lfs iz skripte, pojavljuje se na jos mesta

Comment (lukicdarkoo) [reply install.bash]: E da da

**Review by angstrem98 — DISMISSED** [review]
Galactic here we are :alien:

**Review by angstrem98 — APPROVED** [review]
Galactic here we are :alien: (now for real)

**Pull request merged**